configs/configuration.yaml

```diff
@@ -1,38 +1,73 @@
 config_name: cytosnake_configs
 
-env_manager: mamba
+env_manager: conda
 
 # computation configs
 analysis_configs:
   preprocessing:
     threads: 4
 
-# data configurations
 data_configs:
-  plate_data_format: sqlite
+  # flag indicates to use converted data. if True, then
+  # cytosnake will search for plate datafiles with the extensions set
+  # in `converted_plate_data_format`
+  use_converted_plate_data: False
+
+  # These are the data type configurations
+  # mostly handles on how outputs and inputs should be named or formatted
+  data_types:
+    cell_counts:
+      file_ext: csv
+      suffix: cell_counts
+    plate_data:
+      file_ext: sqlite
+      converted_ext: parquet
+      suffix: None
+      compression_ext: null
+    aggregated:
+      suffix: aggregated
+      file_ext: csv
```

Comment:
This comment applies to the `file_ext` and `compression_ext` for each of these blocks. Do the specifications here limit the ability to use any other file type or compression besides what's specified? Thinking along these lines, what would happen if `npz` or `parquet` types were used?

Reply:
This is dependent on the workflow scripts. If the extensions are supported, it should work. For example, if `.npz` files are being generated, then the user should know that the workflow they have selected (or created) is capable of generating `npz` files. As of right now, I do not know if `pycytominer` can generate `npz` outputs.

However, this brings up a good point for documentation purposes. I should document what are the supported outputs and create a schema that reflects that. 

In addition, I have learned something new. It seems that the `jsonschema` package provides a really nice way on how to check supported parameters by using the `enum` keyword:
```python
import jsonschema

# Define the schema
schema = {
    "type": "object",
    "properties": {
        "fruit": {
            "type": "string",
            "enum": ["apple", "banana", "orange"]
        }
    }
}

# Data to be validated
data = {
    "fruit": "banana"
}

# Validate the data against the schema
try:
    jsonschema.validate(data, schema)
    print("Data is valid!")
except jsonschema.ValidationError as e:
    print("Data is invalid!")
    print(e.message)
``` 

In this example, `banana` is being used under the `fruit` parameter. `banana` is part of the supported parameter words that is reflected on the schema above under the `enum` parameter. If the value of `banana` were to change to `potato`, then an error is raised because it is not a part of the supported value choices under the `enum` section.

This example will greatly help to develop #55